bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
+    while f != os.path.sep:
+        if os.path.ismount(f): return f
+        f = os.path.realpath(os.path.join(f, os.pardir))
+    return f
+
+
+def get_file_system_type(filepath):
+    """
+    Return the file system type the file object file resides on.
+    """
+
+    # normalize,resolve and get absolute path to the file object
```

Comment:
Just as a convention, start comments from a capital letter. A dot at the end is nice too, but I do not remember how strict I was about it in this code. Again, check how other comments are done and mimic the style.